### ISSUE TITLE
error: virtual filesystem overlay file not found

### ISSUE THREAD

**Issue by breferrari** [klarna/klarna-mobile-sdk#73]
Hi 👋 

We use the SDK on our app and we're getting errors in the `lldb` that points paths with your usernames.
This is blocking us to use the `lldb` to properly debug our app. We can't even do a simple `po` command.

```
error: virtual filesystem overlay file '/Users/carlos.jimenez/Library/Developer/Xcode/DerivedData/KlarnaMobileSDK-bfjfbshljhpqyrbnhsmaiszqmfat/Build/Intermediates.noindex/KlarnaMobileSDK.build/Release-iphonesimulator/KlarnaMobileSDK.build/all-product-headers.yaml' not found
error: couldn't IRGen expression. Please check the above error messages for possible root causes.
```

I've found this old issue which is now closed:
https://github.com/klarna/klarna-mobile-sdk/issues/7
Is it related? Was this fixed already?

**Comment by simonpierreroy** [comment]
Seem fixed in 0.29.0

**Comment by gbanfalvikl** [comment]
Hi, yes, there seemed to be an issue in Xcode 12, we've attempted to fix it in 0.29. Let us know if it's still happening?

**Comment by omarbh** [comment]
HI, I will close this ticket, please feel free to open it if you are still experiencing the issue.
